src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -245,9 +243,9 @@ const OwnerContainer = ({ asset }: { asset: WalletAsset }) => {
           {listing ? 'Your Price' : 'List for Sale'}
         </ThemedText.SubHeader>
         {listing && (
-          <ExternalLink href={listing.marketplaceUrl}>
-            <MarketplaceIcon alt={listing.marketplace} src={getMarketplaceIcon(listing.marketplace)} />
-          </ExternalLink>
+          <MarketplaceIcon href={listing.marketplaceUrl}>
+            {getMarketplaceIcon(listing.marketplace, '20', '20', uuidv4())}
```

Comment:
Gradient is looking strange on asset I have for sale on x2y2
<img width="51" alt="Screen Shot 2023-04-24 at 15 19 24 " src="https://user-images.githubusercontent.com/11512321/234128840-934364a0-560a-4828-bdba-1a780b231bfc.png">
<img width="1728" alt="Screen Shot 2023-04-24 at 15 19 20 " src="https://user-images.githubusercontent.com/11512321/234128821-8551292e-a72d-45a6-9854-f2b0f118ad02.png">

Reply:
going to fix did not see that x2y2 icon was using gradients